src/rust/ensogl/src/control/io/mouse.rs

```diff
@@ -109,7 +110,10 @@ macro_rules! define_bindings {
                     }));
                     let js_closure = $name.as_ref().unchecked_ref();
                     let js_name    = stringify!($js_name);
-                    let result     = dom.add_event_listener_with_callback(js_name,js_closure);
+                    let mut options    = AddEventListenerOptions::new();
+                    options.capture(true);
+                    let result     = dom. add_event_listener_with_callback_and_add_event_listener_options_and_wants_untrusted(
+                                      js_name,js_closure,&options,None);
```

Comment:
There should be good docs explaining what is happening here and why. This is complex topic and in case we will revisit it in the future, its worth leaving really proper description here. For example, while doing this review I completely do not understand what this change does and why its needed.